bitnami/dataplatform-bp1/README.md

```diff
@@ -84,107 +84,106 @@ The command removes all the Kubernetes components associated with the chart and
+| `global.imageRegistry`    | Global Docker image registry                    | `nil` |
+| `global.imagePullSecrets` | Global Docker registry secret names as an array | `[]`  |
+| `global.storageClass`     | Global StorageClass for Persistent Volume(s)    | `nil` |
+
 
 ### Zookeeper chart parameters
 
-Parameters below are set as per the recommended values, they can be overwritten if required.
+| Name                                                                                | Description                                                                               | Value  |
+| ----------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ------ |
+| `zookeeper.enabled`                                                                 | Switch to enable or disable the Zookeeper helm chart                                      | `true` |
+| `zookeeper.replicaCount`                                                            | Number of Zookeeper replicas                                                              | `3`    |
+| `zookeeper.heapSize`                                                                | Size in MB for the Java Heap options (Xmx and XMs).                                       | `4096` |
+| `zookeeper.resources.limits`                                                        | The resources limits for Zookeeper containers                                             | `{}`   |
+| `zookeeper.resources.requests`                                                      | The requested resources for Zookeeper containers for a small kubernetes cluster           | `{}`   |
+| `zookeeper.affinity.podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecution` | Zookeeper pods Anti Affinity rules for best possible resiliency (evaluated as a template) | `{}`   |
 
-| Parameter                      | Description                                                                     | Default                                                                              |
-|:-------------------------------|:--------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------|
-| `zookeeper.enabled`            | Switch to enable or disable the Zookeeper helm chart                            | `true`                                                                               |
-| `zookeeper.replicaCount`       | Number of Zookeeper nodes                                                       | `3`                                                                                  |
-| `zookeeper.heap`               | Zookeepers's Java Heap size                                                     | Zookeeper Java Heap size set for optimal resource usage                              |
```

Comment:
⚠️ It looks like this parameter was renamed: `heap` -> `heapSise`

Reply:
The `README.md` value was outdated, in `values.yaml` it was already `heapSize`. What should I do?